explore/src/main/scala/explore/observationtree/obsTree.scala

```diff
@@ -5,26 +5,35 @@ package explore.observationtree
 
 import explore.model.ObsSummary
 import monocle.Lens
-import monocle.macros.Lenses
+import monocle.macros.GenLens
```

Comment:
Maybe we should use `Focus` everywhere

Reply:
Oh, I missed we were using `GenLens`. Is there a difference with `Focus`, or just the new name?